pyodide-build/pyodide_build/common.py

```diff
@@ -12,7 +12,7 @@
 from pathlib import Path
 from typing import NoReturn
 
-import tomli
+import tomllib
```

Comment:
tomlib is a subset of tomli with a compatible API I believe so maybe we could do:
```py
try:
	import tomlib
except ImportError:
	import tomli as tomlib
```
or just continue using `tomli`.

Reply:
I'd always go with using `sys.version_info` instead of try/except, but otherwise this is the standard way to do this. Though you'll need to be a little careful if you run pyupgrade with 3.11+ set, but you shouldn't be doing that anyway on code that's supposed to handle 3.10 too.